test/tests/citeTest.js

```diff
@@ -0,0 +1,48 @@
+describe("Zotero.Cite.System", function() {
+	describe("#retrieveItem()", function() {
+		let article, citeSystem;
+		before(function() {
+			// Load sample item into DB
+			let data = populateDBWithSampleData(loadSampleData('journalArticle'));
+			article = Zotero.Items.get(data.journalArticle.id);
+			
+			// instantiate Zotero.Cite.System object
+			citeSystem = new Zotero.Cite.System();
+		});
+		
+		it('should retrieve item given a Zotero.Item', function() {
+			let cslItem = citeSystem.retrieveItem(article);
+			assert.ok(cslItem);
+			assert.equal(cslItem.id, article.id);
+		});
+		
+		it.skip('should retrieve item given an item URI', function() {
```

Comment:
this is actually not working and I'm not entirely sure what https://github.com/zotero/zotero/blob/4.0/chrome/content/zotero/xpcom/cite.js#L502 is expecting as input. @simonster do you know off the top of your head before I dig into the word processor code?